### PR TITLE
Accessibility improvements to modals

### PR DESCRIPTION
**Pull request by davidbenque**
This PR aims to fix the following issues

- [x] #1537 
- [x] #1561

### CODE REVIEW

**Review by yflory — CHANGES_REQUESTED** [review]

File: www/calendar/app-calendar.less [hunk 1]
```diff
@@ -157,6 +157,11 @@
             border-radius: @variables_radius_L;
         }
         .tui-full-calendar-popup-container {
+            .tui-full-calendar-section-allday{
+                &:focus {
+                    outline: @cryptpad_color_brand solid 2px;
```
[the remark this answers is not in the document]
Comment (dariiing) [reply www/calendar/app-calendar.less:162]: I followed the original behavior of this button: you can click anywhere on the div and get the same result; but I agree it's a bit weird

File: www/calendar/inner.js [hunk 2]
```diff
@@ -2276,6 +2284,7 @@ APP.recurrenceRule = {
                     $b.closest('.tui-full-calendar-floating-layer').hide();
                 });
             });
+            UI.addTabListener(el);
```
[the remark this answers is not in the document]
Comment (dariiing) [reply www/calendar/inner.js:2288]: fixed in https://github.com/cryptpad/cryptpad/pull/1563/commits/6ed9ea3fa331d9ab4107d7271adb7029cbca30c9

File: www/common/common-interface.js [hunk 3]
```diff
@@ -770,7 +771,7 @@ define([
 
         document.body.appendChild(frame);
         setTimeout(function () {
-            $(input).select().focus();
+            addTabListener(frame);
```
[the remark this answers is not in the document]
Comment (dariiing) [reply www/common/common-interface.js:774]: These are the 3 scenarios that I tested:
- focus is on the "Cancel" button and I press the Enter key, the Cancel button triggers
- focus is on the "OK" button and I press the Enter key, the OK button triggers
-  focus is on the name input and I press Enter, the OK button triggers (the same functionality has been tested on prod)
- focus is on the name input and I press Esc, the Cancel button triggers (also tested on prod).
Should I make the Esc work anywhere?

File: www/calendar/app-calendar.less (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidbenque) [reply www/calendar/app-calendar.less:162]: After discussion we agreed to keep the same behaviour as the mouse interaction which highlights the whole "row" on hover

**Review by yflory — APPROVED** [review]

**Pull request merged**